rd-kt/rd-core/src/jvmTest/kotlin/com/jetbrains/rd/util/test/cases/LifetimeTest.kt

```diff
@@ -3,7 +3,6 @@ package com.jetbrains.rd.util.test.cases
 import com.jetbrains.rd.util.lifetime.*
 import com.jetbrains.rd.util.test.framework.RdTestBase
 import com.jetbrains.rd.util.threading.SpinWait
-import org.junit.Test
```

Comment:
Should we consistently use `org.junit.jupiter.api.Test` instead of `kotlin.test.Test` maybe?

Reply:
Well, as I see, we use junit only in jvmTest submodule
I replaced all kotlin.test usages in jvmTest